cmd/ferretdb/main.go

```diff
@@ -125,6 +126,14 @@ var cli struct {
 			Package        string        `default:""                             help:"Telemetry: custom package type."`
 		} `embed:"" prefix:"telemetry-"`
 	} `embed:"" prefix:"test-"`
+
+	OTEL struct {
+		OTLPEndpoint     string        `default:"127.0.0.1:4318" help:"OTLP exporter endpoint."`
```

Comment:
A non-empty value should enable the exporter. Remove all other options.

Reply:
fixed